common/src/test/java/io/netty/util/internal/NativeLibraryLoaderTest.java

```diff
@@ -25,13 +26,56 @@
 
 public class NativeLibraryLoaderTest {
 
+    private static final Method getSupressedMethod = getGetSuppressed();
```

Comment:
ThrowableUtil might have been a better place for this.

Reply:
I thought it was better not to put test-only code (getSuppressed) inside of non- test code (ThrowableUtil), but if that's not a concern I can move it there.